src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -191,15 +200,16 @@ class BankAccountStep extends React.Component {
         // Disable bank account fields once they've been added in db so they can't be changed
         const isFromPlaid = this.props.achData.setupType === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID;
         const shouldDisableInputs = Boolean(this.props.achData.bankAccountID) || isFromPlaid;
+        const {subStep} = this.props.achData;
```

Comment:
Some guidelines on destructuring here - https://github.com/Expensify/App/blob/main/STYLE.md#destructuring

Probably would help to read the entire style guide while you are at it if you haven't yet :)

Reply:
I did! is just that it takes time to remember all rules from a guideline, with time you should have to correct less and less of this type of errors 😬